packages/chat-widget/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "module": "esnext",
+    "module": "nodenext",
```

Comment:
this is the meat of this change. Unclear to me why targeting `esnext` won't pull in `types.d.ts` but it apparently doesn't :-/